wiki/Tournaments/MWC/2020_4K/en.md

```diff
@@ -0,0 +1,223 @@
+---
+tags:
+- MWC 2020
+- MWC 4K 2020
+- MWC2020
+- MWC4K 2020
```

Comment:
```suggestion
- MWC 4K 2020
- MWC2020
- MWC4K 2020
```
should be enough i think

Reply:
Not really needed, it uses past tags updated to the year